languages/csharp/exercises/concept/nullability/.docs/introduction.md

```diff
@@ -0,0 +1,93 @@
+Up to now, we saw how to declare variables and how to assign them to
+values.  For example, the following code snippet declares a variable
+`a` of type `int` whose value is `5`:
+
+```
+int a = 5;
+```
+
```

Comment:
We can assume that the student knows how to declare variables, so we can leave this out.

```suggestion
```